# This file is distributed under the same license as the Django package.
#
msgid ""
msgstr ""
"Project-Id-Version: Django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2011-03-15 13:16-0400\n"
"PO-Revision-Date: 2011-03-09 13:04+0000\n"
"Last-Translator: fasouto <fsoutomoure@gmail.com>\n"
"Language-Team: Galician <None>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: gl\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"

#: db/models/fields.py:50
msgid "The base GIS field -- maps to the OpenGIS Specification Geometry type."
msgstr ""
"O campo GIS base -- correspóndese co tipo Geometry da especificación OpenGIS."

#: db/models/fields.py:270
msgid "Point"
msgstr "Punto"

#: db/models/fields.py:274
msgid "Line string"
msgstr "Secuencia de liñas"

#: db/models/fields.py:278
msgid "Polygon"
msgstr "Polígono"

#: db/models/fields.py:282
msgid "Multi-point"
msgstr "Punto múltiple"

#: db/models/fields.py:286
msgid "Multi-line string"
msgstr "Cadea multi-liña"

#: db/models/fields.py:290
msgid "Multi polygon"
msgstr "Multi polígono"

#: db/models/fields.py:294
msgid "Geometry collection"
msgstr "Colección de xeometrías"

#: forms/fields.py:17
msgid "No geometry value provided."
msgstr "Non se proporcionou un valor de xeometría."

#: forms/fields.py:18
msgid "Invalid geometry value."
msgstr "Valor xeométrico non válido."

#: forms/fields.py:19
msgid "Invalid geometry type."
msgstr "Tipo xeometríco non válido."

#: forms/fields.py:20
msgid ""
"An error occurred when transforming the geometry to the SRID of the geometry "
"form field."
msgstr ""
"Atopouse un erro mentras se trasnformaba a xeometría ó SRID do campo "
"xeométrico do formulario."
